sdk/instrumentation/google.golang.org/grpc/server.go

```diff
@@ -231,3 +237,14 @@ func WrapStatsHandler(delegate stats.Handler, spanFromContext sdk.SpanFromContex
 		dataCaptureConfig: internalconfig.GetConfig().GetDataCapture(),
 	}
 }
+
+func setSchemeAttributes(ctx context.Context, span sdk.Span) {
+	peer, ok := peer.FromContext(ctx)
+	if ok {
```

Comment:
If you do `!ok` this can be turned into early termination and we save one identation.